src/char/inter.c

```diff
@@ -814,19 +814,27 @@ static int inter_config_read(const char* cfgName)
 	return 0;
 }
 
-// Save interlog into sql
+/**
+* Save interlog values to SQL
+* @param fmt: Pointer containing the type of activity to be stored
+*  Supplied arguments: activity, origin, target, guild pointer, castle_id
+* @return 0
+*/
 int inter_log(char* fmt, ...)
```

Comment:
@aleos89 that's actually one of the reasons why I wanted to explicitly pass the values in that function and another thing is that, we're using SQL which is rigid.